module-4/backend/week_one.md

```diff
@@ -1,20 +1,32 @@
-## Week One - Module 4 Recap
+## week_one.md
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+### What's the most useful thing you learned from completing the intermission week work?
+*Java script basics*
+### What are some tools to help debug JavaScript code?
+*console.log*
+*eval(pry.it)*
+### What are some tools you need in order to unit test your JavaScript?
+*install chai and mocha*
+### What is the syntax for invoking a function?
+*myFunction()*
+### What's the difference between == and === in JavaScript?
+*By using == you check if something is equal to something else. This is not strict and by using === its also strict*
+### What's the difference between asynchronous and synchronous JavaScript?
+*asynchronicity is about how to pack up code to execute upon some later event*
```

Comment:
Remember that it executes when the interpreter comes across the line, the interpreter just continues reading without waiting for the function to evaluate to a return value.